sage-ios/SAGE/SAGE/ProfileOperations.swift

```diff
@@ -75,6 +75,10 @@ class ProfileOperations: NSObject {
             userJSON[UserConstants.kPasswordConfirmation] = passwordConfirmation
         }
         
+        if resetPhoto {
```

Comment:
can you put this right below the photoData condition so theyre grouped together?